### PR TITLE
test: fixes health checks for components, sometimes there is an issue…

### PR DESCRIPTION
**Pull request by applike-ss**
… because components are not ready yet;

### CODE REVIEW

File: pkg/test/component_kinesis.go [hunk 1]
```diff
@@ -65,9 +66,16 @@ func doRunKinesis(name string, configMap configInput) {
...
-			_, err := kinesisClient.ListStreams(&kinesis.ListStreamsInput{})
 
-			return err
+			_, err = kinesisClient.ListStreams(&kinesis.ListStreamsInput{})
+
+			if err != nil {
+				return err
+			}
+
+			time.Sleep(1 * time.Second)
```
Comment (josketres) [inline pkg/test/component_kinesis.go]: 😱

File: pkg/test/component_sns.go [hunk 2]
```diff
@@ -85,9 +86,16 @@ func doRunSns(name string, configMap configInput) {
 		},
 		HealthCheck: func() error {
 			snsClient := ProvideSnsClient(name)
-			_, err := snsClient.ListTopics(&sns.ListTopicsInput{})
 
-			return err
+			_, err = snsClient.ListTopics(&sns.ListTopicsInput{})
```
Comment (josketres) [inline pkg/test/component_sns.go]: Did you take a look at the `ListTopicsResponse`?
Maybe it contains something interesting like a "PleaseRetryAgainIn1Second" response.

File: pkg/test/component_kinesis.go (hunk 1 above)
Comment (josketres) [reply pkg/test/component_kinesis.go]: if we really need to add this sleep here maybe we should add a small comment explaining why it was done.

File: pkg/test/component_sns.go (hunk 2 above)
Comment (applike-ss) [reply pkg/test/component_sns.go]: nothing interesting there that we could make use of

**Comment by josketres** [comment]
PR Duplicated.

**Pull request closed without merging**